doc/Language/grammar_tutorial.pod6

```diff
@@ -27,41 +27,75 @@ language.
 
 =SUBTITLE The Broad Concept of Perl Grammars
 
-Regular Expressions (regex) work well to find patterns in strings and manipulate them. However, when you need to find multiple patterns at once, or need to combine patterns, or test for patterns that may surround strings, or other patterns - regular expressions alone are not adequate.
```

Comment:
Opinion accepted! Anyway, even without tests and stuff, this needs to be done by a text formatter, not by hands. I can do it myself with emacs and note this pr in the commit message or you can send another commit here.